content/webinars/verify-your-infrastructure-on-deployment/index.md

```diff
@@ -0,0 +1,83 @@
+---
+# Name of the webinar.
+title: "Verify Your Infrastructure on Each Deployment with Pulumi and CircleCI"
+meta_desc: "Angel Rivera and Chris Smith will walk you through best practices for app developers, DevOps, and SREs to keep configuration mistakes from reaching production."
+
+# A featured webinar will display first in the list.
+featured: false
+
+# If the video is pre-recorded or live.
+pre_recorded: false
+
+# If the video is part of the PulumiTV series. Setting this value to true will list the video in the "PulumiTV" section.
+pulumi_tv: false
+
+# The preview image will be shown on the list page.
+preview_image: "https://circleci.com/blog/media/CircleCI+Pulumi.png"
```

Comment:
We should probably not do this and just pull the image into our site.

Reply:
agreed - the image was an example -- if we need to download or ask for high-res, I'm happy to do it.

Reply:
I've copied it into this PR, yep.